src/main/java/com/cinema/Main.java

```diff
@@ -118,8 +119,11 @@ public static void main(String[] args) throws AuthenticationException {
         AuthenticationService authenticationService = (AuthenticationService) injector
                 .getInstance(AuthenticationService.class);
         authenticationService.register("roman@mail.ru", "roman");
-        logger.info("Login user :" + authenticationService.login("roman@mail.ru", "roman"));
-
+        try {
+            authenticationService.login("roman@mail.ru", "roman");
+        } catch (AuthenticationException e) {
+            logger.error("Login error. Incorrect userEmail or password!", e);
```

Comment:
error is too much, I think
nothing really bad happened - `warn` level will be enough

Reply:
done